core/templates/components/state-directives/response-header/response-header.component.html

```diff
@@ -70,7 +69,7 @@
     cursor: pointer;
     display: inline-block;
     max-width: 100px;
-    overflow: hidden;
+    overflow: inherit;
```

Comment:
Could you please explain why this is updated?

Reply:
I have done some css change to make the UI as per the requirement of the issue during this i change over-flow. But now done as previous . thanks for mentioning.